cookbooks/logs.rst

```diff
@@ -61,6 +69,35 @@ includes language errors such as PHP Errors, Warnings, and Notices, as well as
 uncaught exceptions. It also contains your application logs if you log on
 ``stderr``.
 
+Because SymfonyCloud manages this file for you (preventing disks to get filled
+and using very fast local drives instead of slower network disk), we recommend
+that applications always output their log to ``stder``.
+
+.. _monolog-configuration-stderr:
+
+.. note::
+
+   For developers convenience and for historic reasons, Monolog default recipe
```

Comment:
historical?

Reply:
that's what I wrote initially but then I read again Fabien's blogpost (https://symfony.com/blog/logging-in-symfony-and-the-cloud) where he used "historic" 🤷

Reply:
that's a typo :) historical reasons is the right way.